smooth_component/CMakeLists.txt

```diff
@@ -20,10 +20,13 @@ set(CMAKE_CXX_STANDARD 17)
 include(${COMPONENT_DIR}/../lib/compiler_options.cmake)
 include(${COMPONENT_DIR}/../lib/files.cmake)
 
+set(_idf_path $ENV{IDF_PATH})
```

Comment:
I can't stand warts on variable names (such as prefix `_`, `m_`, `g_` etc.) and much prefer written out names. Please rename to `normalized_path` and its all good.

Reply:
Done